src/wallet.js

```diff
@@ -50,6 +50,16 @@ import _ from 'lodash';
  * @namespace Wallet
  */
 const wallet = {
+  /*
+   * Should never be accessed directly, only through get method
+   */
+  _rewardSpendMinBlocks: null,
+
+  /*
+   * Should never be accessed directly, only through get method
+   */
+  _networkHeight: 0,
```

Comment:
I think we need a better definition of what it means. Is it the highest height (even if the block is not in the best blockchain) or is it the height of the best blockchain's head?

Reply:
Updated the variable name to `_networkBestChainHeight` and add a comment explaining it

Reply:
Great! Thanks!